.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Build master document
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```

このリポジトリのデフォルトブランチは `main` です